tests/e2e/support/fixtures.ts

```diff
@@ -0,0 +1,34 @@
+import { test as base } from '@playwright/test'
+
+import { type Fixture, deployFixture } from './deploy-to-netlify'
+
+/* eslint-disable-next-line @typescript-eslint/no-empty-interface */
```

Comment:
Why did we want this empty interface around anyhow?

Reply:
It serves as documentation. The `base.extend<T, W>` API is really inexpressive. I thought this made this more obvious.

What's funny is I considered removing it due to the need for an eslint-disable, but it turns out I'd still need an eslint-disable for something else (passing `{}` as the first type param) anyway, so I kept it.